media/js/base/stub-attribution.js

```diff
@@ -434,8 +491,12 @@ if (typeof window.Mozilla === 'undefined') {
             ua: ua,
             experiment: experiment,
             variation: variation,
-            client_id: clientID,
-            session_id: clientID ? StubAttribution.createSessionID() : null,
+            client_id: clientIDUA,
+            client_id_ga4: clientIDGA4,
```

Comment:
If you're adding a new field to stub attribution then you also need to add it to the list of known fields in bedrock's back-end stub authentication service (this exists so that people can't encode any suspicious data) https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/views.py#L46

This is probably why you don't see the new field when decoding what bedrock's back-end is returning.

Reply:
Working! Thanks!